packages/ddmarlinpandora/package.py

```diff
@@ -46,13 +46,11 @@ class Ddmarlinpandora(CMakePackage, Ilcsoftpackage):
 
     variant("monitoring", default=False, description="Enable Pandora Monitoring")
 
-    def setup_build_environment(self, env):
-        if "+monitoring" in self.spec:
-            env.set("PANDORA_MONITORING", "ON")
-
     def setup_run_environment(self, env):
         env.prepend_path("MARLIN_DLL", self.prefix.lib + "/libDDMarlinPandora.so")
 
     def cmake_args(self):
-        # C++ Standard
-        return [f"-DCMAKE_CXX_STANDARD={self.spec['root'].variants['cxxstd'].value}"]
+        return [
+            f"-DCMAKE_CXX_STANDARD={self.spec['root'].variants['cxxstd'].value}",
```

Comment:
Does this work with c++20? Or are we fixing https://github.com/PandoraPFA/PandoraSDK/pull/18 this some other way?

Reply:
Yeah this works, for a while we have been building DDMarlinPandora with C++20. For PandoraSDK it's still C++17 since it's hardcoded in the recipe...

Reply:
OK. Then it probably was related to another Pandora package that includes the header and stumbles. Just wanted to make sure here.